src/sass/base/_typography.scss

```diff
@@ -1,37 +1,10 @@
-//* These styles impact size, weight, letter-spacing, etc of the fonts on the page! 
+//* GLOBAL text/font-face/type settings go here. Things like links as they display across all parts of the page 
+//! Remember, consistency and bad design is better than good design and no consistency 
 
-// Nav typography
-
-.navItem {
-    color: white;
-    font-size: 2.4rem;
-    font-family: $main-type;
```

Comment:
I removed these styles to the nav.scss partial - I did this because these font styles weren't GLOBAL, they were just applying to the navbar at the end of the day.